src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py

```diff
@@ -152,6 +152,7 @@ def connection_create(cmd, client,  # pylint: disable=too-many-locals
                       user_identity_auth_info=None, system_identity_auth_info=None,
                       service_principal_auth_info_secret=None,
                       key_vault_id=None,
+                      service_endpoint=None,
```

Comment:
why only service endpoint? what about other vnet scenarios? private link?

Reply:
suppose there will be some description to redirect user to private link solution right

Reply:
more description added